tests/python_on_whales/components/test_compose.py

```diff
@@ -469,6 +479,20 @@ def test_compose_run_detach():
     assert container.logs() == "dodo\n"
 
 
+def test_docker_compose_run_labels():
+    container = docker.compose.run(
+        "alpine",
+        ["echo", "dodo"],
+        labels={"traefik.enable": "false"},

```

Comment:
Can you also add a test with more than one label? To make sure the syntax is ok.

Reply:
Done!